### PR TITLE
MOB-408: [iOS SDK] Get CPU usage and call stack for specific thread

### PR DESCRIPTION
**Pull request by tomisacat**
Issue task: https://wiredcraft.atlassian.net/browse/MOB-408

### CODE REVIEW

File: iOS/DoraemonKit/Src/Core/Plugin/Performance/CPU/Function/DoraemonCPUUtil.m [hunk 1]
```diff
@@ -24,23 +30,32 @@ + (CGFloat)cpuUsageForApp {
...
             // cpu_usage : Scaled cpu usage percentage. The scale factor is TH_USAGE_SCALE.
             //宏定义TH_USAGE_SCALE返回CPU处理总频率：
-            tot_cpu += basic_info_th->cpu_usage / (float)TH_USAGE_SCALE;
+            float usage = basic_info_th->cpu_usage / (float)TH_USAGE_SCALE;
+            usageModel.totalUsage += usage;
+            
+            if (usage > usageModel.maxUsageOfThread) {
+                NSString *callstack = [DoraemonBacktraceLogger doraemon_backtraceOfThread:thread_list[j]];
+                if (callstack.length > 0) {
+                    usageModel.maxUsageOfThread = usage;
```
Comment (Tdreamy) [inline iOS/DoraemonKit/Src/Core/Plugin/Performance/CPU/Function/DoraemonCPUUtil.m:55]: - the callstack by DoraemonBacktraceLogger  is no symbolization
- we give data to react,  the callStack don't need the all callstack,  maybe can give a func to get the main callstack

Comment (tomisacat) [reply iOS/DoraemonKit/Src/Core/Plugin/Performance/CPU/Function/DoraemonCPUUtil.m:55]: * Any suggestions on getting symbolized call stack?
* I think it's out of the task, maybe we should have a discussion on it.

File: iOS/DoraemonKit/Src/Core/Plugin/Performance/CPU/Function/DoraemonCPUUtil.m [hunk 2]
```diff
@@ -8,11 +8,17 @@
 #import "DoraemonCPUUtil.h"
 #import <mach/mach.h>
 #import <UIKit/UIKit.h>
+#import "DoraemonBacktraceLogger.h"
 //#import <sys/sysctl.h>
 
 @implementation DoraemonCPUUtil
 
 + (CGFloat)cpuUsageForApp {
+    DoraemonCPUUsageCallStackModel *model = [self cpuUsageAndCallstackForThread];
```
Comment (Tdreamy) [inline iOS/DoraemonKit/Src/Core/Plugin/Performance/CPU/Function/DoraemonCPUUtil.m:17]: The cpu samples at a rate of 0.5 seconds， we don't need  fetch call stack with each sample,  get callstack  when cpu usage is anomaly，

Comment (tomisacat) [reply iOS/DoraemonKit/Src/Core/Plugin/Performance/CPU/Function/DoraemonCPUUtil.m:17]: sure.

**Pull request closed without merging**